whirlpool/dryer.py

```diff
@@ -0,0 +1,288 @@
+ATTR_CYCLE_STATUS_SENSING = "DryCavity_CycleStatusSensing"
+ATTR_CYCLE_STATUS_STATICREDUCE = "DryCavity_CycleStatusStaticReduce"
+ATTR_CYCLE_STATUS_STEAMING = "DryCavity_CycleStatusSteaming"
+ATTR_CYCLE_STATUS_WET = "DryCavity_CycleStatusWet"
+
+# DrySys
+ATTR_OP_SET_DAMPNOTIFICATIONTONEVOLUME = "DrySys_OpSetDampNotificationToneVolume" 
+
+# Sys
+ATTR_OP_SET_ALERTTONEVOLUME = "Sys_OpSetAlertToneVolume"
+
+# XCat
+ATTR_ODOMETER_STATUS_CYCLECOUNT = "XCat_OdometerStatusCycleCount"
+ATTR_ODOMETER_STATUS_RUNNINGHOURS = "XCat_OdometerStatusRunningHours"
+ATTR_ODOMETER_STATUS_TOTALHOURS = "XCat_OdometerStatusTotalHours"
+
+ATTR_WIFI_STATUS_ISPCHECK = "XCat_WifiStatusIspCheck"
+ATTR_WIFI_STATUS_RSSIANTENNADIVERSITY = "XCat_WifiStatusRssiAntennaDiversity"
+
+
+class MachineState(Enum):
+    Standby = 0
+    Setting = 1
+    DelayCountdownMode = 2
+    DelayPause = 3
+    SmartDelay = 4
+    SmartGridPause = 5
+    Pause = 6
+    RunningMainCycle = 7
+    RunningPostCycle = 8
+    Exceptions = 9
+    Complete = 10
+    PowerFailure = 11
+    ServiceDiagnostic = 12
+    FactoryDiagnostic = 13
+    LifeTest = 14
+    CustomerFocusMode = 15
+    DemoMode = 16
+    HardStopOrError = 17
+    SystemInit = 18
+    Mystery = 19
+
+
+MACHINE_STATE_MAP = {
+    MachineState.Standby: ATTRVAL_MACHINE_STATE_STANDBY,
+    MachineState.Setting: ATTRVAL_MACHINE_STATE_SETTING,
+    MachineState.DelayCountdownMode: ATTRVAL_MACHINE_STATE_DELAYCOUNTDOWNMODE,
+    MachineState.DelayPause: ATTRVAL_MACHINE_STATE_DELAYPAUSE,
+    MachineState.SmartDelay: ATTRVAL_MACHINE_STATE_SMARTDELAY,
+    MachineState.SmartGridPause: ATTRVAL_MACHINE_STATE_SMARTGRIDPAUSE,
+    MachineState.Pause: ATTRVAL_MACHINE_STATE_PAUSE,
+    MachineState.RunningMainCycle: ATTRVAL_MACHINE_STATE_RUNNINGMAINCYCLE,
+    MachineState.RunningPostCycle: ATTRVAL_MACHINE_STATE_RUNNINGPOSTCYCLE,
+    MachineState.Exceptions: ATTRVAL_MACHINE_STATE_EXCEPTIONS,
+    MachineState.Complete: ATTRVAL_MACHINE_STATE_COMPLETE,
+    MachineState.PowerFailure: ATTRVAL_MACHINE_STATE_POWERFAILURE,
+    MachineState.ServiceDiagnostic: ATTRVAL_MACHINE_STATE_SERVICEDIAGNOSTIC,
+    MachineState.FactoryDiagnostic: ATTRVAL_MACHINE_STATE_FACTORYDIAGNOSTIC,
+    MachineState.LifeTest: ATTRVAL_MACHINE_STATE_LIFETEST,
+    MachineState.CustomerFocusMode: ATTRVAL_MACHINE_STATE_CUSTOMERFOCUSMODE,
+    MachineState.DemoMode: ATTRVAL_MACHINE_STATE_DEMOMODE,
+    MachineState.HardStopOrError: ATTRVAL_MACHINE_STATE_HARDSTOPORERROR,
+    MachineState.SystemInit: ATTRVAL_MACHINE_STATE_SYSTEMINIT,
+}
+
+
+class Dryer(Appliance):
+    def get_machine_state(self) -> MachineState | None:
+        state_raw = self._get_attribute(ATTR_MACHINE_STATE)
+        for k, v in MACHINE_STATE_MAP.items():
+            if v == state_raw:
+                return k
+        return None
+
+    def get_machine_state_value(self) -> int | None:
+        return self._get_int_attribute(ATTR_MACHINE_STATE)
+
+    def get_op_status_dooropen(self) -> bool | None:
```

Comment:
```suggestion
    def get_door_open(self) -> bool | None:
```

lets also rename the other methods to make them simpler, like the ATTR consts suggested above